src/Revit/RevitHyparTools/WallByProfile.cs

```diff
@@ -1,40 +1,34 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Elements;
 using Elements.Geometry;
 using Elements.Geometry.Solids;
 
-namespace Hypar.Revit {
-    [UserElement]
-    public class WallByProfile : Wall
+namespace Elements {
+    public partial class WallByProfile 
     {
-        public Line CenterLine {get;}
-        public double Thickness {get;}
-        public WallByProfile(Line centerLine,
-                             Profile profile,
-                             double thickness,
-                             Material material = null,
-                             Transform transform = null,
-                             Representation representation = null,
-                             bool isElementDefinition = false,
-                             Guid id = default(Guid),
-                             string name = null) : base(transform != null ? transform : new Transform(),
-                                                 material != null ? material : BuiltInMaterials.Concrete,
-                                                 representation != null ? representation : new Representation(new List<SolidOperation>()),
-                                                 isElementDefinition,
-                                                 id != default(Guid) ? id : Guid.NewGuid(),
-                                                 name)
-        {
-            Profile = profile;
-            CenterLine = centerLine;
-            Thickness = thickness;
+        public WallByProfile(Profile @profile, double @thickness, Line @centerline, Transform @transform=null, Material @material=null, Representation @representation=null, bool @isElementDefinition=false)
+            : base(transform != null ? transform : new Transform(),
+                   material != null ? material : BuiltInMaterials.Default,
```

Comment:
I think our walls use `BuiltInMaterials.Concrete` by default. If not, they should.

Reply:
Why don't we just add this to Elements proper so anybody using elements can take advantage of it?